SalesforceCommerceCloud/pwa-kit
# Fix: Add keywords meta tag to Seo component

# Description



Seo components adds the meta tag if the keywords prop is present.
This pr addresses this issue https://github.com/SalesforceCommerceCloud/pwa-kit/issues/1761

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# How to Test-Drive This PR

- start template retail app
- check document <head> for keywords meta tag


# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
